### PR TITLE
feat: upload add download icon

### PR DESCRIPTION
**Pull request by crazyair**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [x] TypeScript definition update
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   upload  add download icon      |
| 🇨🇳 Chinese |     upload 添加下载按钮     |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by qq645381995** [comment]
语言包只加了中文和英文

**Comment by qq645381995** [comment]
@afc163 会考虑合并吗，添加了下载 Icon，删除 Icon 改用了 delete（我觉得不是关闭，而是删除所以才改的）

**Comment by qq645381995** [comment]
@afc163 这都没人看嘛？还是没时间？

**Comment by afc163** [comment]
feature branch

**Comment by qq645381995** [comment]
终于 ok 了，

**Review by afc163 — CHANGES_REQUESTED** [review]
feature branch please

File: components/upload/index.zh-CN.md [hunk 1]
```diff
@@ -33,13 +33,14 @@ title: Upload
 | multiple | 是否支持多选文件，`ie10+` 支持。开启后按住 ctrl 可选择多个文件 | boolean | false |  |
 | name | 发到后台的文件参数名 | string | 'file' |  |
 | previewFile | 自定义文件预览逻辑 | (file: File \| Blob) => Promise<dataURL: string> | 无 | 3.17.0 |
-| showUploadList | 是否展示文件列表, 可设为一个对象，用于单独设定 `showPreviewIcon` 和 `showRemoveIcon` | Boolean or { showPreviewIcon?: boolean, showRemoveIcon?: boolean } | true |  |
+| showUploadList | 是否展示文件列表, 可设为一个对象，用于单独设定 `showPreviewIcon` 和 `showRemoveIcon` 和 `showDownloadIcon` | Boolean or { showPreviewIcon?: boolean, showRemoveIcon?: boolean, showDownloadIcon?: boolean } | true |  |
```
Comment (MrHeer) [inline components/upload/index.zh-CN.md]:
```diff
- | showUploadList | 是否展示文件列表, 可设为一个对象，用于单独设定 `showPreviewIcon` 和 `showRemoveIcon` 和 `showDownloadIcon` | Boolean or { showPreviewIcon?: boolean, showRemoveIcon?: boolean, showDownloadIcon?: boolean } | true |  |
+ | showUploadList | 是否展示文件列表, 可设为一个对象，用于单独设定 `showPreviewIcon`, `showRemoveIcon` 和 `showDownloadIcon` | Boolean or { showPreviewIcon?: boolean, showRemoveIcon?: boolean, showDownloadIcon?: boolean } | true |  |
```

**Review by MrHeer — CHANGES_REQUESTED** [review]

**Comment by qq645381995** [comment]
下一步要做啥？不太懂

**Comment by afc163** [comment]
https://ant.design/docs/react/contributing-cn#%E5%88%86%E6%94%AF%E7%AE%A1%E7%90%86

**Comment by qq645381995** [comment]
> https://ant.design/docs/react/contributing-cn#%E5%88%86%E6%94%AF%E7%AE%A1%E7%90%86

https://github.com/ant-design/ant-design/pull/18661

**Pull request closed without merging**